control-plane/tests/test_cli_inspection.py

```
@@ -589,6 +590,272 @@ def __init__(self, server_address: tuple[str, int], handler_class: type) -> None
+            review_owner="analyst-001",
+            intended_outcome="Review repository owner change evidence before any approval-bound response.",
+        )
+        base_now = datetime.now(timezone.utc)
+        expires_at = base_now + timedelta(hours=4)
+
+        with self.assertLogs("aegisops.control_plane", level="INFO") as log_output:
+            service.ingest_wazuh_alert(
+                raw_alert=_load_wazuh_fixture("github-audit-alert.json"),
+                authorization_header="Bearer reviewed-shared-secret",
+                forwarded_proto="https",
+                reverse_proxy_secret_header="reviewed-proxy-secret",
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to argument: "reverse_proxy_secret_header"

(S106)